app/util/object-store.ts

```diff
@@ -212,6 +228,20 @@ export class S3ObjectStore {
     return params;
   }
 
+  /**
+   * Downloads the given object from the store, returning a string containing the contents of the
+   * object
+   *
+   * @param paramsOrUrl - a map of parameters (Bucket, Key) indicating the object to
+   *   be retrieved or the object URL
+   * @returns a string containing the contents of the object
+   * @throws TypeError - if an invalid URL is supplied
+   */
+  async download(paramsOrUrl: string | BucketParams): Promise<string> {
+    const getObjectResponse = this.getObject(paramsOrUrl);
+    return streamToString(getObjectResponse.createReadStream());
```

Comment:
Don't you need to `await` on this call?

Reply:
No, since `streamToString` also returns a `Promis<string>`.